packages/unimodules-permissions-interface/package.json

```diff
@@ -28,8 +28,11 @@
   "author": "650 Industries, Inc.",
   "license": "MIT",
   "homepage": "https://github.com/expo/expo/tree/master/packages/unimodules-permissions-interface",
+  "unimodulePeerDependencies": {
+    "@unimodules/core": "*"
+  },
```

Comment:
we don't need this, actually we should remove this field from all unimodules' package.json